administration/config/webpackDevServer.config.ts

```diff
@@ -32,6 +35,7 @@ function createWebpackDevServerConfig(proxy: Configuration['proxy'], allowedHost
     // really know what you're doing with a special environment variable.
     // Note: ["localhost", ".localhost"] will support subdomains - but we might
     // want to allow setting the allowedHosts manually for more complex setups
+    // eslint-disable-next-line no-nested-ternary
```

Comment:
Can't the nested ternary be avoided? Would be nicer

Reply:
Since this is just a configuration file and copied from another project, I did not feel like spending too much time on this. But yes, they could be avoided. I think its okay here though.